### ISSUE TITLE
cmake error  libevent-2.1.11-stable 

### ISSUE THREAD

**Issue by tan00** [libevent/libevent#914]
The release   `libevent-2.1.11-stable.tar.gz`   miss file  `Uninstall.cmake.in` , and cmake get error

[release    tar link](https://github.com/libevent/libevent/releases/download/release-2.1.11-stable/libevent-2.1.11-stable.tar.gz)

```
CMake Error: File /home/lin/src/libevent-2.1.11-stable/cmake/Uninstall.cmake.in does not exist.
CMake Error at CMakeLists.txt:1498 (configure_file):
  configure_file Problem configuring file


-- 
           ---( Libevent 2.1.11-stable )---
-- 
-- Available event backends: EPOLL;SELECT;POLL
-- CMAKE_BINARY_DIR:         /home/lin/src/libevent-2.1.11-stable/build
-- CMAKE_CURRENT_BINARY_DIR: /home/lin/src/libevent-2.1.11-stable/build
-- CMAKE_SOURCE_DIR:         /home/lin/src/libevent-2.1.11-stable
-- CMAKE_CURRENT_SOURCE_DIR: /home/lin/src/libevent-2.1.11-stable
-- PROJECT_BINARY_DIR:       /home/lin/src/libevent-2.1.11-stable/build
-- PROJECT_SOURCE_DIR:       /home/lin/src/libevent-2.1.11-stable
-- CMAKE_MODULE_PATH:        /home/lin/src/libevent-2.1.11-stable/cmake/
-- CMAKE_COMMAND:            /usr/bin/cmake
-- CMAKE_ROOT:               /usr/share/cmake-3.7
-- CMAKE_SYSTEM:             Linux-4.4.0-17763-Microsoft
-- CMAKE_SYSTEM_NAME:        Linux
-- CMAKE_SYSTEM_VERSION:     4.4.0-17763-Microsoft
-- CMAKE_SYSTEM_PROCESSOR:   x86_64
-- CMAKE_SKIP_RPATH:         NO
-- CMAKE_VERBOSE_MAKEFILE:   FALSE
-- CMAKE_C_FLAGS:             -Wall -Wextra -Wno-unused-parameter -Wstrict-aliasing -Wstrict-prototypes -fno-strict-aliasing -Wmissing-prototypes -Winit-self -Wmissing-field-initializers -Wdeclaration-after-statement -Waddress -Wnormalized=id -Woverride-init -Wlogical-op -Wwrite-strings
-- CMAKE_BUILD_TYPE:         Release
-- CMAKE_C_COMPILER:         /usr/bin/cc (id GNU, clang 0, GNUC 1)
-- CMAKE_AR:                 /usr/bin/ar
-- CMAKE_RANLIB:             /usr/bin/ranlib
-- 
-- Configuring incomplete, errors occurred!
See also "/home/lin/src/libevent-2.1.11-stable/build/CMakeFiles/CMakeOutput.log".
See also "/home/lin/src/libevent-2.1.11-stable/build/CMakeFiles/CMakeError.log".
```

**Comment by tan00** [comment]
Fixed  by add  `Uninstall.cmake.in`   copy from github

**Comment by azat** [comment]
Thanks for the report, it has already been fixed in upstream in f05ba6719, and it will be backported to the future releases.
